tutor/templates/apps/openedx/settings/cms/common.py

```diff
@@ -9,10 +9,16 @@
 VIDEO_TRANSCRIPTS_SETTINGS['STORAGE_KWARGS']['location'] = MEDIA_ROOT
 
 # Change syslog-based loggers which don't work inside docker containers
-LOGGING['handlers']['local'] = {'class': 'logging.NullHandler'}
+LOGGING['handlers']['local'] = {
+    'class': 'logging.handlers.WatchedFileHandler',
+    'filename': '/openedx/logs/edx.log',
```

Comment:
I propose to rename this to `all.log`: because the logs will be persisted in `data/SERVICENAME/logs`, the "Open edX" name is already stored in the log file path, on the host.

I'm not being very clear... Do you understand my point?